plugins/modules/opkg.py

```diff
@@ -58,7 +58,6 @@
             - "remove"
             - "checksum"
             - "removal-of-dependent-packages"
-        default: ""
```

Comment:
Please document that `""` and not specifying this parameter both means that no `--force` parameter is used.

Reply:
Done